CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# dbt_zendesk v0.12.1
```

Comment:
Given the possibility of the source being a breaking change, let's also make this a breaking change to reflect that update.